ledger-core/v2/insolar/payload/vcallrequestflags_test.go

```diff
@@ -0,0 +1,30 @@
+// Copyright 2020 Insolar Network Ltd.
+// All rights reserved.
+// This material is licensed under the Insolar License version 1.0,
+// available at https://github.com/insolar/assured-ledger/blob/master/LICENSE.md.
+
+package payload
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
```

Comment:
you do need something like 
TestCallRequestFlags_BitStorageIsEngough
that ensures, that bit storage is enough to fit your flags

Reply:
https://github.com/insolar/assured-ledger/pull/229/files#diff-cf3d0de3817dffe38e53f19bf2bf656eR16-R23 why that is not enough?

Reply:
fair enough